src/main/java/com/stripe/net/LiveStripeResponseGetter.java

```diff
@@ -149,8 +149,8 @@ protected PasswordAuthentication getPasswordAuthentication() {
 		} else {
 			conn = (HttpURLConnection) stripeURL.openConnection();
 		}
-		conn.setConnectTimeout(30 * 1000);
-		conn.setReadTimeout(80 * 1000);
+		conn.setConnectTimeout(options.getConnectionTimeout());
```

Comment:
What do you think about keeping the naming of `Connect` instead of adding `Connection`? Besides just consistency, I think that the former is more accurate given that it's a verb (like read), and the timeout is the maximum time allowed as that action is being taken.